docs/architecture/grpc-for-wcf-developers/types-of-rpc.md

```diff
@@ -7,5 +7,258 @@ ms.date: 09/02/2019
 
```

Comment:
I'd suggest we name this article rpc-types.md to avoid unnecessary small words (another one of our rules)